build_scripts/build_usd.py

```diff
@@ -1078,6 +1200,41 @@ def InstallJPEG_Turbo(context, force, buildArgs):
         extraJPEGArgs = buildArgs
         if MacOS():
             extraJPEGArgs.append("-DWITH_SIMD=FALSE")
+        if context.targetIos:
+            extraJPEGArgs.append('-DCMAKE_SYSTEM_PROCESSOR=aarch64')
+            extraJPEGArgs.append("-DENABLE_STATIC=TRUE")
+
+            # Replace test and utility executables with static libraries to avoid issues with code signing.
```

Comment:
Instead of fixing tj for them, can we instead suppress the creation of test and utility executables for tj, as they are unrelated to building and deploying USD. It makes sense to me to upstream fixes to tj overall.

Reply:
Disabled openimagiio for the time being - OpenEXR needs it.